src/components/Navbar.js

```diff
@@ -13,7 +13,8 @@ import MenuItem from "@mui/material/MenuItem";
 import { Link, useNavigate } from "react-router-dom";
 import { useAuth } from "../contexts/AuthContexts";
 import { Alert } from "react-bootstrap";
-
+import PersonIcon from '@mui/icons-material/Person';
```

Comment:
Could you surround imports with Brackets just to keep consistent with the style of the file?

Reply:
I agree. Also a good practice to reserve double quotes for HTML, so at some point, let's change those to single quote marks for the imports. I'll do a more formal review well when I get the chance!

Reply:
> Could you surround imports with Brackets just to keep consistent with the style of the file?

Got it, thank you!